<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ramakrishnan Periyakaruppan</author>
 <class>DialogOpenCdc</class>
 <widget class="QDialog" name="DialogOpenCdc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose CDC and Xap.xml</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/images.qrc">
    <normaloff>:/new/images/cdc.png</normaloff>:/new/images/cdc.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Choose configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCdcPath">
        <property name="text">
         <string>CDC Path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="cdcPath"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="browseCDC">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelXapPath">
        <property name="text">
         <string>Xap.xml Path</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="xapPath">
        <property name="placeholderText">
         <string>Xap.xml is expected to be present on the CDC path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="message"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
